teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorKeysOptions.java

```diff
@@ -120,6 +121,12 @@ public void configure(TekuConfiguration.Builder builder) {
                     convertToPath(validatorExternalSignerTruststorePasswordFile)));
   }
 
+  private List<String> parseValidatorExternalKeys() {
+    return validatorExternalSignerPublicKeys.stream()
+        .filter(val -> !Strings.isNullOrEmpty(val))
```

Comment:
I'd suggest using the commons lang3 `StringUtils.isNotBlank` here which will also filter out strings that are entirely whitespace. You also won't need to negate it so can use a method reference.